features/FxA-80-eye-password-experiment/README.md

```diff
@@ -0,0 +1,124 @@
+
+Eye Password Experiment
+========================
+
+https://mozilla.aha.io/features/FXA-80
+
+Firefox Account users are struggling immensely when changing their passwords.
+
+![Change Password Statistics](change-password-stats.png)
```

Comment:
It would be great to include a deep link to the actual graph in datadog here, or if that's not possible, a description of precisely which graph it is so we can dig down in datadog and find it.  Just to be sure we'll all talking about the same thing.